tools/explorer/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ ExternalProject_Add(
 
 add_custom_target(explorer
   COMMENT "Building tt-explorer... ${TTMLIR_BIN_DIR}"
-  COMMAND pip install ${CMAKE_CURRENT_SOURCE_DIR}/tt_adapter
+  COMMAND pip install -e ${CMAKE_CURRENT_SOURCE_DIR}/tt_adapter
```

Comment:
Intentional?

Reply:
Yeah, it makes development a lot easier if the module is auto updating. I can make this only true on `Debug` instead of `Release` in case that works better.